src/lib/operators/table_scan/abstract_table_scan_impl.hpp

```diff
@@ -25,17 +25,18 @@ class AbstractTableScanImpl {
    */
```

Comment:
> Binary size for the TPC-H grows from 35M to 45M
As in, the size of the release build binary? Were compile times similarly affected?

Reply:
Yes and no. It's just padding that gets added.